ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11blend.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "Direct3D11Headers.h"
 #include "Graphics_Systems/General/GSblend.h"
+#include "Graphics_Systems/General/GScolors.h"
```

Comment:
Kind of silly but it appears in some previous pull request we put `currentblendmode` in `GScolors.h` instead of `GSblend.h`. Since I removed the redundant extern for `currentblendmode` in this source, I had to include the header which contains the declaration. I can still move `currentblendmode` from `GScolors.h` to `GSblend.h` before I merge this.

Reply:
Ok: 60ef6a4a59156a3325082a4bf08be4cfb8dbd068

I gave blend state its own generic source and moved the blend state declaration to its header for you. Now the blend sources don't need those color includes.